include/swift/AST/ExtInfo.h

```diff
@@ -604,31 +694,41 @@ class SILExtInfoBuilder {
 
   // Note that we don't have setters. That is by design, use
   // the following with methods instead of mutating these objects.
+  LLVM_NODISCARD
```

Comment:
In the past, the `withXYZ` methods on `AnyFunctionType::ExtInfo`and `SILFunctionType::ExtInfo` were marked `LLVM_NODISCARD` presumably because they were "immutable update methods", so it doesn't make sense to call the methods and ignore the result. There was an oversight by me when refactoring the code to use the builder-style API, where some `withXYZ` methods were marked `LLVM_NODISCARD` but not others. I was fixing that oversight.

Reply:
Ah, okay.